sharding-jdbc/sharding-jdbc-core/src/main/java/org/apache/shardingsphere/shardingjdbc/jdbc/core/connection/ShadowConnection.java

```diff
@@ -93,13 +102,14 @@ public PreparedStatement prepareStatement(final String sql, final String[] colum
     
     @Override
     public void setAutoCommit(final boolean autoCommit) throws SQLException {
+        this.autoCommit = autoCommit;
         actualConnection.setAutoCommit(autoCommit);
         shadowConnection.setAutoCommit(autoCommit);
     }
     
     @Override
     public boolean getAutoCommit() {
-        return false;
+        return autoCommit;
```

Comment:
Can we use lombok's @Getter to instead of it?

Reply:
getAutoCommit()、isClosed()、isReadOnly() extends from Connection interface , can not use 
 lombok's @getter to instead of them.